modules/csharp/pages/gs-install.adoc

```diff
@@ -58,10 +68,38 @@ This is the recommended method of dependency management because it supports the
 * `Couchbase.Lite.Support.NetDesktop`
 * `Couchbase.Lite.Support.WinUI`
 
+=== Installing Vector Search 
+
+Couchbase recommends installing Nuget packages via _PackageReference_.
+
+[IMPORTANT]
+--
+To use Vector Search, you must have Couchbase Lite installed and add the Vector Search extension to your Couchbase Lite application.
+Vector Search is available only for 64-bit architectures and
+Intel processors that support the Advanced Vector Extensions 2 (AVX2) instruction set.
+To verify whether your device supports the AVX2 instructions set, https://www.intel.com/content/www/us/en/support/articles/000090473/processors/intel-core-processors.html[follow these instructions.] 
+--
+
+==== Package Reference
+This is the recommended method of dependency management because it supports the strict version requirement between the core _Couchbase Lite for .Net_ package and its dependent Support library, which comprises:
+
+.`Couchbase.Lite.Enterprise`
+* `Couchbase.Lite.Enterprise.Support.UWP`
+* `Couchbase.Lite.Enterprise.Support.ios`
+* `Couchbase.Lite.Enterprise.Support.android`
+* `Couchbase.Lite.Enterprise.Support.NetDesktop`
+* `Couchbase.Lite.Enterprise.Support.WinUI`
+
+.`Couchbase.Lite.VectorSearch`
+* `Couchbase.Lite.VectorSearch.Support.UWP`
+* `Couchbase.Lite.VectorSearch.Support.ios`
+* `Couchbase.Lite.VectorSearch.Support.android`
+* `Couchbase.Lite.VectorSearch.Support.NetDesktop`
+* `Couchbase.Lite.VectorSearch.Support.WinUI`
```

Comment:
I don't think this list is correct. @borrrden Can you confirm?

Reply:
As @borrrden reminded me on slack about this, I think the package will just be "Couchbase.Lite.VectorSearch" and there are no Support packages needed to be installed.

Reply:
Okay I've removed the support packages in this section for Couchbase.Lite.VectorSearch. I'm assuming you still need the Enterprise packages for CBL so have left them as is.